presentations/2023-08-02_mlcsu-ksn-meeting/index.qmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "Travels with R and Python - the power of data science in healthcare"
+subtitle: "[Chris Beeley](mailto:chris.beeley1@nhs.net)"
+format:
+  revealjs:
+    theme: [default, ./su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    logo: logo_yellow.svg
+    slide-number: false
+    auto-animate: true
+---
+
+[ds_presentations]: https://the-strategy-unit.github.io/data_science/presentations/2023-08-02_mlcsu-ksn-meeting
+
+## What is data science?
+
+* "A data scientist knows more about computer science than the average statistician, and more about statistics than the average computer scientist"
```

Comment:
Source? Looks like it came from a Medium blog https://medium.com/odscjournal/data-scientists-versus-statisticians-8ea146b7a47f#:~:text=%E2%80%9C%20A%20data%20scientist%20is%20someone,statistics%2C%20and%20specific%20domain%20knowledge which is blocked on our VPN so I can't read if it's a quote from somewhere else.

Reply:
Done